notebooks/Solution_Exercises.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8c3882b9",
+   "metadata": {},
+   "source": [
+    "# Solutions of exercises\n",
+    "\n",
+    "Exercises are available in a few notebooks. Below are solutions to the exercises.\n",
+    "How to use the notebook:\n",
+    "\n",
+    "* **DO NOT RUN THE NOTEBOOK**\n",
+    "* **Copy the code snippet(s) and run the snippet(s) in the relevant notebook**\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "38dce9f9",
+   "metadata": {},
+   "source": [
+    "## Exercises from notebook idr0062_prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "75db4ae2",
+   "metadata": {},
+   "source": [
+    "### Exercise 1 "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47d19053",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Question 1: Read the geojson file:\n",
+    "import json\n",
+    "with open(geojson_file) as f:\n",
+    "    d = json.load(f)\n",
+    "    s = json.dumps(d)\n",
+    "values = geojson.loads(s)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8fa7860",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Question 2: Overlay the shapes from a specific plane on the image\n",
+    "z = 5\n",
```

Comment:
Sorry, one more inclarity. Is ``5`` somehow preferred ? I mean, if I change the value of ``z`` ot any other number than ``5``, I am not getting a red shape on my image. Only in case I choose ``5`` I do get a red shape. What am I missing ?
![Screenshot 2022-04-05 at 16 40 15](https://user-images.githubusercontent.com/2478303/161792352-bf310ad7-e810-4b3a-9581-f337f9d9f20f.png)
![Screenshot 2022-04-05 at 16 40 03](https://user-images.githubusercontent.com/2478303/161792359-d9225a2d-dcde-4f64-9bca-f5f00e040d2e.png)

Reply:
It should not matter but it seems that only part of the file is loaded. I will have to check

Reply:
> part of the file is loaded.

Yes, if I instert 

``print (len(values))``

the answer is ``2``.

Reply:
Indeed, the problem is that the ``len(values))`` is giving too low a number. But if I hardcode  ``149`` as below, I get the expected ROI as on the screenshot, so even the higher array members are present. It is just that the iteration stops too early, because of the length of the array is somehow not interpreted correctly ?


![Screenshot 2022-04-05 at 17 00 53](https://user-images.githubusercontent.com/2478303/161796743-45a2946c-c68d-4979-9d2e-7ee4e8c44514.png)

Reply:
fixed